<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrecipitatePhaseWidget</class>
 <widget class="StatsGenWidget" name="PrecipitatePhaseWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>903</width>
    <height>633</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <widget class="QLabel" name="label_8">
     <property name="styleSheet">
      <string notr="true">QLabel
{
font-weight: bold;
font-size: 150%;
}</string>
     </property>
     <property name="text">
      <string>Precipitate Phase</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="sizeDistributionTab">
      <attribute name="title">
       <string>Size Distribution</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>4</number>
       </property>
       <property name="topMargin">
        <number>8</number>
       </property>
       <property name="rightMargin">
        <number>4</number>
       </property>
       <property name="bottomMargin">
        <number>4</number>
       </property>
       <item>
        <widget class="StatsGenFeatureSizeWidget" name="m_FeatureSizeDistWidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Preset Statistic Models</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="microstructurePresetCombo">
           <property name="enabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="m_GenerateDefaultData">
           <property name="text">
            <string>Create Data</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="omega3DistributionTab">
      <attribute name="title">
       <string>Omega 3 Distributions</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="leftMargin">
        <number>4</number>
       </property>
       <property name="topMargin">
        <number>4</number>
       </property>
       <property name="rightMargin">
        <number>4</number>
       </property>
       <property name="bottomMargin">
        <number>4</number>
       </property>
       <item>
        <widget class="StatsGenPlotWidget" name="m_Omega3Plot" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="shapeDistributionTab">
      <attribute name="title">
       <string>Shape Distributions</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="leftMargin">
        <number>4</number>
       </property>
       <property name="topMargin">
        <number>4</number>
       </property>
       <property name="rightMargin">
        <number>4</number>
       </property>
       <property name="bottomMargin">
        <number>4</number>
       </property>
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>879</width>
            <height>600</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="leftMargin">
            <number>4</number>
           </property>
           <property name="topMargin">
            <number>4</number>
           </property>
           <property name="rightMargin">
            <number>4</number>
           </property>
           <property name="bottomMargin">
            <number>4</number>
           </property>
           <item>
            <widget class="StatsGenPlotWidget" name="m_BOverAPlot" native="true"/>
           </item>
           <item>
            <widget class="StatsGenPlotWidget" name="m_COverAPlot" native="true"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="rdfTab">
      <attribute name="title">
       <string>Radial Distribution Function</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="leftMargin">
        <number>4</number>
       </property>
       <property name="topMargin">
        <number>4</number>
       </property>
       <property name="rightMargin">
        <number>4</number>
       </property>
       <property name="bottomMargin">
        <number>4</number>
       </property>
       <item>
        <widget class="StatsGenRDFWidget" name="m_RdfPlot" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="StatsGenODFWidget" name="m_ODFWidget">
      <attribute name="title">
       <string>ODF</string>
      </attribute>
     </widget>
     <widget class="StatsGenAxisODFWidget" name="m_AxisODFWidget">
      <attribute name="title">
       <string>Axis ODF</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StatsGenWidget</class>
   <extends>QWidget</extends>
   <header location="global">StatsGenWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>StatsGenPlotWidget</class>
   <extends>QWidget</extends>
   <header location="global">StatsGenPlotWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>StatsGenODFWidget</class>
   <extends>QWidget</extends>
   <header location="global">StatsGenODFWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>StatsGenAxisODFWidget</class>
   <extends>QWidget</extends>
   <header location="global">StatsGenAxisODFWidget.h</header>
  </customwidget>
  <customwidget>
   <class>StatsGenRDFWidget</class>
   <extends>QWidget</extends>
   <header location="global">StatsGenRDFWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>StatsGenFeatureSizeWidget</class>
   <extends>QWidget</extends>
   <header>StatsGenFeatureSizeWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
